src/app/api/services/guards/auth-guard.service.ts

```diff
@@ -11,10 +11,11 @@ import { DotRouterService } from './../dot-router-service';
 export class AuthGuardService implements CanActivate {
     constructor(private dotRouterService: DotRouterService, private loginService: LoginService) {}
 
-    canActivate(): Observable<boolean> {
+    canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
```

Comment:
Do we need to pass both params if we are not using them?

Reply:
Yes, because we are using the the second one, state.